app/src/main/java/org/oppia/app/testing/HtmlParserTestActivity.kt

```diff
@@ -22,7 +22,7 @@ class HtmlParserTestActivity : InjectableAppCompatActivity() {
     val rawDummyString =
       "\u003cp\u003e\"Let's try one last question,\" said Mr. Baker. \"Here's a pineapple cake cut into pieces.\"\u003c/p\u003e\u003coppia-noninteractive-image alt-with-value=\"\u0026amp;quot;Pineapple cake with 7/9 having cherries.\u0026amp;quot;\" caption-with-value=\"\u0026amp;quot;\u0026amp;quot;\" filepath-with-value=\"\u0026amp;quot;pineapple_cake_height_479_width_480.png\u0026amp;quot;\"\u003e\u003c/oppia-noninteractive-image\u003e\u003cp\u003e\u00a0\u003c/p\u003e\u003cp\u003e\u003cstrong\u003eQuestion 6\u003c/strong\u003e: What fraction of the cake has big red cherries in the pineapple slices?\u003c/p\u003e"
     val htmlResult: Spannable =
-      htmlParserFactory.create( /* entityType= */ "exploration", /* entityId= */ "oppia")
+      htmlParserFactory.create( /* entityType= */ "exploration", /* entityId= */ "oppia", /* imageCenterAlign= */ false)
```

Comment:
@veena14cs Is this correct?

Reply:
it should be true here. As it is checking content.